src/common/transformers/TRIM.spec.js

```diff
@@ -0,0 +1,25 @@
+import expect from 'expect';
+
+import { trimString } from './TRIM';
+
+describe('TRIM', () => {
+    it('should return trim value', () => {
+        expect(trimString(' hello  ')).toBe('hello');
+    });
+
+    it('should return trim value of each item in array', () => {
+        expect(trimString([' hello   ', '\nworld\r\n'])).toEqual([
+            'hello',
+            'world',
+        ]);
+    });
+
+    it('should return null if value is literal', () => {
```

Comment:
if value is NOT a literal?

Reply:
no, the text is right, { a: 'hello ', b: 'world ' } is a literal. But maybe `is not a string` would be even better.